edge/pkg/metamanager/process.go

```diff
@@ -671,16 +671,17 @@ func (m *metaManager) runMetaManager() {
 		for {
 			select {
 			case <-beehiveContext.Done():
-				klog.Warning("MetaManager mainloop stop")
+				klog.Warning("MetaManager main loop stop")
 				return
 			default:
 			}
-			if msg, err := beehiveContext.Receive(m.Name()); err == nil {
-				klog.V(2).Infof("get a message %+v", msg)
-				m.process(msg)
-			} else {
+			msg, err := beehiveContext.Receive(m.Name())
+			if err != nil {
 				klog.Errorf("get a message %+v: %v", msg, err)
+				continue
 			}
```

Comment:
```suggestion
			if msg, err := beehiveContext.Receive(m.Name()); err != nil {
				klog.Errorf("get a message %+v: %v", msg, err)
				continue
			}
```

Reply:
variable `msg` need to be processed when receive succeed, so cannot use the suggested format.